src/assets/stylesheets/Instructions.scss

```diff
@@ -3,23 +3,25 @@
 @use "../../../node_modules/@raspberrypifoundation/design-system-core/scss/mixins/typography";
 @use "./rpf_design_system/colours" as *;
 @use "./rpf_design_system/spacing" as *;
-@use "./rpf_design_system/font-size" as *;
-@use "./rpf_design_system/font-weight" as *;
 
 .project-instructions {
   block-size: 100%;
 
   h2 {
-    @include font-size-1-25(bold);
+    @include typography.style-1-5(bold);
```

Comment:
Does this change the size of the `h2` at all?

Reply:
Ah I see, the variables had changed between `v0.0.x` and `v0.1.0` of the design system 😅 👍